torch/onnx/_internal/exporter/_verification.py

```diff
@@ -40,8 +40,8 @@ class VerificationInfo:
     def from_tensors(
         cls,
         name: str,
-        expected: torch.Tensor | int | float | bool,
-        actual: torch.Tensor | int | float | bool,
+        expected: torch.Tensor | torch.types.Number,
+        actual: torch.Tensor | torch.types.Number,
```

Comment:
Is this a new torch based tool?

Reply:
Do we need this to be elsewhere in this file as well?

Reply:
I found it referenced somewhere else. `torch.types.Number` is only an alias of the type `int | float | bool` so it's just for readability.